src/app_service/service/chat/service.nim

```diff
@@ -303,7 +307,7 @@ QtObject:
       discard status_chat.deactivateChat(chatId)
 
       self.chats.del(chatId)
-      discard status_chat.clearChatHistory(chatId)
+      discard status_chat.deleteMessagesByChatId(chatId)
```

Comment:
This looks funny, but what's going on here is that `clearChatHistory` actually called the `deleteMessagesByChatID` RPC API (which is desired here, but the naming of the method is incorrect (which caused me to reimplement clear history syncing in status-go while it already exists..)

Luckily, there was also already a `deleteMessagesByCHatId` method, which is doing the **correct** thing, so this change here, is not a breaking change.